packages/now-client/README.md

```diff
@@ -50,16 +50,18 @@ Full list of events:
 
 ```js
 [
-  // File events (receive relevant data as payload)
+  // File events
   'hashes-calculated',
+  'file-count',
   'file-uploaded',
   'all-files-uploaded',
-  // Deployment events (receive deployment object as payload)
+  // Deployment events
   'created',
+  'building',
   'ready',
+  'alias-assigned',
+  'warning',
   'error',
```

Comment:
Is there a master list somewhere in the API? It would be great if we could share that object to somehow keep in sync (might not be possible because it’s probably a separate monorepo though)

Reply:
I realize this is the readme so i really meant to comment on the Set() below 😇

Reply:
No, those events only come from `now-client`, the deployment creates different events that do not match with those. Those are just easier for a client to process.